tests/ontopy_tests/test_patch.py

```diff
@@ -44,37 +41,36 @@
 assert emmo.Atom["altLabel"] == ["ChemicalElement"]
 
 
+assert emmo.Atom.is_defined == False
```

Comment:
Is_defined should be tested as part of test_ontopy as it is defined there.

Reply:
We are testing the `is_defined` property of Atom, which is patched. The implementation in ontology is deprecated.